### ISSUE TITLE
Support for 1D Barcodes

### ISSUE THREAD

**Issue by werthdavid** [werthdavid/ngx-scanner#52]
After merging the [PR #12](https://github.com/zxing-js/library/pull/12) in the zxing-library, we will be able to detect 1D Barcodes as well. We should make the scanner fit for that as well 
* extending BrowserBarcodeReader if needed
* at least update Wiki and Readme

**Comment by odahcam** [comment]
So, I would go with a new option, like: `1d`, `2d`,  `both` for performance reasons. So we could select the appropriated reader class in the `BrowserBarcodeReader`. 🤔

**Comment by werthdavid** [comment]
Yepp! I'm using ZXing on Android/Java, there you have constants for the different barcode-types and you can then define which codes are to be detected

**Comment by odahcam** [comment]
News: https://github.com/zxing-js/library/pull/12 has merged!

**Comment by susannaHayrapetyan** [comment]
Hi, when we can use this feature? is it available now? I see it's not working for demo apps you have provided in readme.

**Comment by werthdavid** [comment]
Can we close this now @odahcam ?

**Comment by odahcam** [comment]
I don't think so, because I didn't implemented anything here to use the new library's reader. Let's wait until our first 1D barcode read. :)
